components/coding/studentPortal/radix/DialogComponent.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
```

Comment:
Can we rewrite this component so it takes in any child component as a prop? This way it will be reusable for several other use cases.

I'm also thinking we should have some sort of general prop to control the size. Sizing looks pretty solid for the intro page banner, but thinking about other use cases. cc @LuckyHariharan - who wants to use this type of component for the badge details "page"